appeals/web/src/server/appeals/appeal-details/appellant-case/address/address.controller.js

```diff
@@ -74,6 +75,13 @@ export const postChangeSiteAddress = async (request, response) => {
 		return response.redirect(redirectUrl);
 	} catch (error) {
 		logger.error(error);
+
+		// Check if it's a validation error (400)
```

Comment:
Would it be worth moving this check into a named utility function? Just a thought